TourMate/TourMate/TripViews/TripView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TripDetailView.swift
+//  Tourmate
+//
+//  Created by Rayner Lim on 7/3/22.
+//
+
+import SwiftUI
+
+struct TripView: View {
+    @EnvironmentObject var model: MockModel
+
+    @State var id: Int
+    @State private var isActive = false
+
+    var body: some View {
+        let trip = model.trips[id]
+
+        return ItineraryView(id: id)
+            .navigationTitle(trip.name)
+            .toolbar {
+                ToolbarItem(placement: .primaryAction) {
+                    NavigationLink(isActive: $isActive) {
```

Comment:
Maybe the Plans' FormViews are having error because of the navigation link? E.g. changing the `isActive` to false caused the trip view to re-render then the children views to also re-render multiple times. Is there another way to clear the stack? Tbh not very sure about how the stack works maybe can discuss